### ISSUE TITLE
Add "open all

### ISSUE THREAD

**Issue by bjherbison** [mozilla-mobile/fenix#10878]

**Comment by astarq** [comment]
In addition it would be useful if, instead of the static manner in which tabs are currently selected, one could long press a tab and scroll up or down like in a gallery app, or certain chromium browsers.

**Comment by AmyYLee** [comment]
Assigned @topotropic for consideration

**Comment by yoasif** [comment]
Ran into this due to trying to migrate tabs from an old Fenix on an old device and finding that https://github.com/mozilla-mobile/fenix/issues/11404 does not exist, nor does https://github.com/mozilla-mobile/fenix/issues/2080, so I opened my tabs on desktop, bookmarked them, and tried to use this (missing) feature.

**Comment by AmyYLee** [comment]
@topotropic Does this need further UX investigation?

**Comment by topotropic** [comment]
No, doesn't require UX investigation – sounds like a valid use case. Thanks for raising it. 

Let's add the option to open all pages that are part of a bookmark folder to the 3-dot menu of the folder. 
@betsymi currently we use "Open tabs" in Collections to open all and "Open all in tabs" on desktop for the same action. Can you make a recommendation on the copy here? Thanks!

**Comment by kbrosnan** [comment]
Closing as a duplicate of https://github.com/mozilla-mobile/fenix/issues/11404
